arbos/parse_l2.go

```diff
@@ -467,25 +503,39 @@ func parseBatchPostingReportMessage(rd io.Reader, chainId *big.Int, msgBatchGasC
 	}), nil
 }
 
+func GetEspressoJstBytes(jst *arbostypes.EspressoBlockJustification) ([]byte, error) {
+	var result []byte
+	jstJson, err := json.Marshal(jst)
```

Comment:
It's not obvious to me why we have to Json serialize before RLP serialization so would appreciate a comment.